packages/plugins/privacy/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "@ninetailed/experience.js-plugin-privacy",
   "version": "7.6.0-beta.1",
+  "type": "commonjs",
```

Comment:
Why do we use commonjs here while all other packages are type module?

Reply:
It's actually not the only one. 

NX@16 is no longer adding the `type` field to all the generated packages. 
To enforce having the `type` field in all packages we add it manually based on these two criteria:

- We either match what we have in the generated packages prior to the NX@16 upgrade.
- If the package doesn't have an `esm`  build, we give it a type of `commonjs`.